src/data/finsupp/basic.lean

```diff
@@ -1281,6 +1281,23 @@ lemma multiset_sum_sum [has_zero M] [add_comm_monoid N] {f : α →₀ M} {h : 
   multiset.sum (f.sum h) = f.sum (λa b, multiset.sum (h a b)) :=
 (multiset.sum_add_monoid_hom : multiset N →+ N).map_sum _ f.support
 
+section disjoint_prod_add
+variables [add_comm_monoid M]
```

Comment:
Now that this is just one lemma, I think there's no need for a section containing it.
```suggestion
```